ui/src/app/features/model-table/model-table.component.html

```diff
@@ -16,21 +16,23 @@
 
 <table *ngIf="!isLoading" mat-table [dataSource]="data">
 
+  <caption></caption>
```

Comment:
please add a meaningful description inside the caption tag. don't forget to use translations